### ISSUE TITLE
Deploy artifact to SNAPSHOT repository

### ISSUE THREAD

**Issue by KeithYokoma** [mercari/RxReduxK#28]
Currently, builds fail for uploading duplicate artifacts on master branch workflow.
It seems to be better to upload them as a SNAPSHOT, and upload the finalized artifact to the release repository when we create a new tag.

**Comment by kittinunf** [comment]
That is correct. The slight problem was that last time I checked Binary doesn't support SNAPSHOT artifact without the enterprise account. :'(
